pkg/apis/compliance/v1alpha1/complianceremediation_types.go

```diff
@@ -181,6 +213,14 @@ func RemediationWasCreatedByOperator(obj metav1.Object) bool {
 	return ok
 }
 
+func ParseRemediationDependencyRefs(rawdeps string) ([]RemediationObjectDependencyReference, error) {
```

Comment:
Do you think this function's input would ever be something else than the annotation? If not, it might be nicer to make it a method. (for tests, we could use an internal function that just takes a string).
btw it might be better to also check the rawdeps for `!=""` if just return an empty list rather than an unrecoverable error in case the function ever gets `""` as input.

Reply:
good point, I'll change it